docs/source/usage/01_usage.md

```diff
@@ -68,6 +68,7 @@ are shared among multiple roles:
 - `cifmw_nolog`: (Bool) Toggle `no_log` value for selected tasks. Defaults to `true` (hiding those logs by default).
 - `cifmw_parent_scenario`: (String or List(String)) path to existing scenario/parameter file to inherit from.
 - `cifmw_configure_switches`: (Bool) Specifies whether switches should be configured. Computes in `reproducer.yml` playbook. Defaults to `false`.
+- `cifmw_use_ocp_overlay`: (Boolean) Specifies whether devscripts deployed OCP nodes should use overlay images. Defaults to `false`.
```

Comment:
I'd rather NOT change the default behaviour. QE and other ppl not wanting the overlay should really make it clear in their env (i.e. toggling that boolean).

Reply:
Done